### PR TITLE
DR 44 Update Readme

### PR DESCRIPTION
**Pull request by cheesehero112**
### Summary of Changes

- Updated README according to the new work in progress Frontend doc in Notion
- Uploaded a screenshot of the landing page

---

### Summary of Implementation Logic

- Screenshot image URL will be added after the PR is approved.
---

### Checklist for PR Owner

- [x] Standardized PR Title
- [ ] Included relevant videos and screen shots
- [x] Included Summary of Changes
- [x] Included Summary of Implementation Logic
- [ ] Acceptance Criteria are fulfilled

### CODE REVIEW

**Comment by cheesehero112** [comment]
@RubySpeeders I did not copy over everything from the Notion FE doc, because I wasn't sure how we want to format this README to include both FE & BE info.  I'm not sure if there is a word limit to README - Is the goal to make this README identical to the Notion docs or make this one more concise and provide links to the detailed doc in Notion?

File: README.md [hunk 1]
```diff
@@ -1,12 +1,14 @@
-# Musician Personnel Assignment Project
+# Drum Roulette Project
```
[the remark this answers is not in the document]
Comment (cheesehero112) [reply README.md:1]: sounds good!

**Comment by cheesehero112** [comment]
@RubySpeeders made changes!

File: README.md [hunk 2]
```diff
@@ -90,16 +92,53 @@ This command will format all files in the current directory and its subdirectori
...
+- Library Imports
+- Type/Interface Imports
+- Other Component Imports
+- Styles or CSS Imports
+- Constants or Config Imports
+
+## Development Workflow & Contributing
 
-Contributions are welcome! If you'd like to contribute to this project, please make sure to follow the code formatting guidelines defined by Prettier. This ensures consistent code style and makes it easier to review and merge your contributions. If you'd like to contribute to the Musician Personnel Assignment Project, please follow these guidelines:
+Contributions are welcome! If you'd like to contribute to this project, please make sure to follow the code formatting guidelines defined by Prettier. This ensures consistent code style and makes it easier to review and merge your contributions. If you'd like to contribute to the Drum Roulette, please follow these guidelines:
```
Comment (RubySpeeders) [inline README.md:106]: rogue "the" before Drum Roulette! After that is removed, I think ready for merging!

Comment (cheesehero112) [reply README.md]: @RubySpeeders changes are committed! Thanks for your review.

Comment (cheesehero112) [reply README.md]: FYI - After this is merged, I will then need to use the screenshot image URL from Github and add to the README.

**Review by RubySpeeders — APPROVED** [review]

**Pull request merged**